src/main/java/startsteps/ECommerceShop/exeptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package startsteps.ECommerceShop.exeptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler(UserNotFoundException.class)
+    public ResponseEntity<Object> handleUserNotFoundException(UserNotFoundException exception){
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
+                .body(exception.getMessage());
```

Comment:
let's make all responses JSONs instead of string.

Reply:
done